logging/cluster-logging-release-notes.adoc

```diff
@@ -17,16 +17,20 @@ include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
 |====
 
 // Release Notes by version
+include::modules/cluster-logging-release-notes-5.3.z.adoc[leveloffset=+1]
+
 [id="cluster-logging-release-notes-5-3-0"]
 == OpenShift Logging 5.3.0
-The following advisories are available for OpenShift Logging 5.3.x:
+This release includes link:https://access.redhat.com/errata/RHSA-2021:4627[RHSA-2021:4627 OpenShift Logging Bug Fix Release 5.3.0]
 
 [id="openshift-logging-5-3-0-new-features-and-enhancements"]
 === New features and enhancements
 * With this update, authorization requirements for Log Forwarding have been relaxed. Outputs may now be configured with SASL, username/password, or TLS.
 
 [id="openshift-logging-5-3-0-bug-fixes"]
 === Bug fixes
+* Before this update records shipped via syslog would serialize a ruby hash encoding key/value pairs to contain a '=>' character, as well as replace tabs with "#11".  This update serializes the message correctly as proper JSON. (link:https://issues.redhat.com/browse/LOG-1494[LOG-1494])
```

Comment:
```suggestion
* Before this update, if you forwarded logs using the syslog protocol, serializing a ruby hash encoded key/value pairs to contain a '=>' character and replaced tabs with "#11".  This update fixes the issue so that log messages are correctly serialized as valid JSON. (link:https://issues.redhat.com/browse/LOG-1494[LOG-1494])
```
WDYT? 
With suggestion, I'm trying to do the following: 
- Avoids using "would." IBM SG: "Use only to refer to an unfulfilled condition or hypothesis and to express the future in the past"
- Helps make the precondition, forwarding using syslog, clearer.